### DISCUSSION TITLE
Port this to Linux.

### DISCUSSION THREAD

**Discussion by LAPIII** [ventoy/Ventoy#2757] [Ideas]
Ventoy is such an awesome utility that I recommend it to people, but many people use Linux. I would like to know if there are any plans to port this.

**Comment by Perkolator** [comment]
What are you talking about? Port what to linux? It's been available to use in linux over 4 years already. Do you mean the GUI, not terminal?

> [For convenience, Ventoy has provided native GUI program in Linux system since 1.0.52.](https://www.ventoy.net/en/doc_linux_gui.html)

Version 1.0.52 was released on Sep 13, 2021.

**Reply by GuilhermeNobrega** [reply]
Hey man... we are talking about ventoy! is like..

"Ventoy is an open source tool to create bootable USB drive for ISO/WIM/IMG/VHD(x)/EFI files.
With ventoy, you don't need to format the disk over and over, you just need to copy the image files to the USB drive and boot it."

https://www.ventoy.net/en/index.html
https://github.com/ventoy/Ventoy

**Reply by Perkolator** [reply]
??? I don't understand what you're trying to say. Also, I was answering to the OP, not to your reply.. in case there was a misunderstanding. :)

But, I see that you're also talking about "porting" to linux (on March 2024) although Ventoy has been **officially** working on linux for many years. So, what "porting" are you two talking about?

**Reply by Perkolator** [reply]
> While it originally supported Windows systems primarily, Ventoy has since added official support for Linux.

Where do you get this information? It's completely false. Ventoy has officially supported linux from the very first release (CLI, later GUI too). Since the github repo releases do not have the first versions, one can check the validity of my claim from [this page](https://www.ventoy.net/cn/doc_start.html) which (only the chinese language version of that page) mentions a `ventoy-1.0.00-linux.tar.gz` file. The 1.0.00 version was the first release of Ventoy. I really don't understand what you are talking about.

**Reply by Perkolator** [reply]
Your first post on Mar 21 you say that Ventoy does NOT officially support linux. Then your response to me you said that Ventoy initially supported only Windows and later added linux support. ALL FALSE.
